src/app/workbench/WorkbenchView.tsx

```diff
@@ -1,21 +1,31 @@
 import * as React from 'react';
 import { IDType, isVisynDataViewDesc, isVisynSimpleViewDesc, useAsync, ViewUtils } from 'tdp_core';
+import { MosaicBranch, MosaicPath } from 'react-mosaic-component';
 import { useMemo } from 'react';
 import { IWorkbenchView } from '../../store';
 import { WorkbenchGenericView } from './WorkbenchGenericView';
 import { WorkbenchEmptyView } from './WorkbenchEmptyView';
 import { useAppSelector } from '../../hooks/useAppSelector';
 
-export interface IWorkbenchViewProps {
-  workbenchIndex: number;
-  view: IWorkbenchView;
-}
-
 export function getVisynView(entityId: string) {
   return ViewUtils.findVisynViews(new IDType(entityId, '.*', '', true));
 }
 
-export function WorkbenchView({ workbenchIndex, view }: IWorkbenchViewProps) {
+export function WorkbenchView({
+  workbenchIndex,
+  view,
+  dragMode,
+  path,
+  setMosaicDrag,
+  removeCallback,
+}: {
+  workbenchIndex: number;
+  view: IWorkbenchView;
+  dragMode: boolean;
+  path: MosaicBranch[];
+  setMosaicDrag: (b: boolean) => void;
```

Comment:
```suggestion
  setMosaicDrag: (isDragging: boolean) => void;
```
I suggest to use a self-explanatory variable name. Please change at other occurrences, too.

Reply:
removed all occurrences of this function anyways for reasons stated above. But agreed!